.github/workflows/build-image.sh

```diff
@@ -93,9 +94,5 @@ for v in 8 11 17; do
         fi
     fi
 done
-#if [ "$PUSH" == "true" ]; then
```

Comment:
Was this supposed to be removed?

Reply:
It was an attempt to save space before the DB2 and Oracle integration tests were moved to separate jobs.